app/controllers/fitbit_profiles_controller.rb

```diff
@@ -9,7 +9,7 @@ def index
     @fitbit_profiles = FitbitProfile
       .includes(:user)
       .order("#{sort_column} #{sort_direction}")
-      .paginate(page: params[:page], per_page: 20)
+      .paginate(page: params[:page], per_page: 15)
```

Comment:
Hound only comments on what you changed - in this case lines 10 to 12 should be aligned. It's always good to obey the Hound, so that we have a consistent as possible codebase :)